brainglobe_utils/pandas/misc.py

```diff
@@ -25,14 +41,24 @@ def safe_pandas_concat(df1: pd.DataFrame, df2: pd.DataFrame) -> pd.DataFrame:
     """
     Concatenate two DataFrames without relying on deprecated functionality
     when one of the DataFrames is empty.
+
     If df1 and df2 are non-empty, return the concatenation.
```

Comment:
```suggestion
    If df1 and df2 are non-empty, return the concatenation. df2 will extend df1 in the resulting DataFrame.
```

Something I should have added for clarity when writing this function :sweat_smile: